### PR TITLE
cleanup some NEXT_MAJOR comments

### PR DESCRIPTION
**Pull request by dmaicher**
## Subject

This is a start to clean-up `NEXT_MAJOR` comments on master.

I am targeting this branch, because we need to prepare master for releasing 4.0.

### CODE REVIEW

File: src/Guesser/TypeGuesser.php [hunk 1]
```diff
@@ -24,50 +24,29 @@
...
-        'orm_many_to_one' => TemplateRegistry::TYPE_MANY_TO_ONE,
-        'orm_one_to_one' => TemplateRegistry::TYPE_ONE_TO_ONE,
-        'number' => TemplateRegistry::TYPE_FLOAT,
-    ];
-
     public function guessType(string $class, string $property, ModelManagerInterface $modelManager): ?TypeGuess
     {
         if (!$ret = $this->getParentMetadataForProperty($class, $property, $modelManager)) {
-            return new TypeGuess(TemplateRegistry::TYPE_STRING, [], Guess::LOW_CONFIDENCE);
+            return new TypeGuess(FieldDescriptionInterface::TYPE_STRING, [], Guess::LOW_CONFIDENCE);
```
Comment (dmaicher) [inline src/Guesser/TypeGuesser.php:30]: Will wait for https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1283 to be merged into master first

File: src/Admin/FieldDescription.php [hunk 2]
```diff
@@ -18,64 +18,48 @@
 /**
  * @final since sonata-project/doctrine-orm-admin-bundle 3.24
```
Comment (core23) [inline src/Admin/FieldDescription.php:19]: You can also remove this comments

**Comment by dmaicher** [comment]
One functional test is failing because of

```
SELECT o FROM Sonata\DoctrineORMAdminBundle\Tests\App\Entity\Author o WHERE o.street LIKE :address_street_0

  at /home/runner/work/SonataDoctrineORMAdminBundle/SonataDoctrineORMAdminBundle/vendor/doctrine/orm/lib/Doctrine/ORM/Query/QueryException.php:43
  at Doctrine\ORM\Query\QueryException::dqlError('SELECT o FROM Sonata\\DoctrineORMAdminBundle\\Tests\\App\\Entity\\Author o WHERE o.street LIKE :address_street_0')
     (/home/runner/work/SonataDoctrineORMAdminBundle/SonataDoctrineORMAdminBundle/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:506)
  at Doctrine\ORM\Query\Parser-&gt;semanticalError('line 0, col 78 near \'street LIKE \': Error: Class Sonata\\DoctrineORMAdminBundle\\Tests\\App\\Entity\\Author has no field or association named street', array('value' =&gt; 'street', 'type' =&gt; 102, 'position' =&gt; 78))
     (/home/runner/work/SonataDoctrineORMAdminBundle/SonataDoctrineORMAdminBundle/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:779)
  at Doctrine\ORM\Query\Parser-&gt;processDeferredPathExpressions()
     (/home/runner/work/SonataDoctrineORMAdminBundle/SonataDoctrineORMAdminBundle/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:284)
  at Doctrine\ORM\Query\Parser-&gt;getAST()
     (/home/runner/work/SonataDoctrineORMAdminBundle/SonataDoctrineORMAdminBundle/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:372)
  at Doctrine\ORM\Query\Parser-&gt;parse()
  ``` 
  
  any idea? seems like something is wrong with a fieldname mapping or so

**Comment by dmaicher** [comment]
ready for review now with smaller changeset

File: tests/Functional/EmbeddedMappingTest.php [hunk 3]
```diff
@@ -28,6 +28,8 @@ public function testFilterByEmbedded(): void
             'filter[address__street][value]' => 'Mancha',
         ]);
 
+        echo $this->client->getCrawler()->html();
```
Comment (VincentLanglet) [inline tests/Functional/EmbeddedMappingTest.php:31]: Echo :)

**Review by VincentLanglet — APPROVED** [review]

**Review by franmomu — APPROVED** [review]
To debug functional tests locally you can execute something like:

```bash
php -S localhost:8000 -t tests/App/public
```

**Pull request merged**